.cirrus.yml

```diff
@@ -0,0 +1,34 @@
+env:
```

Comment:
Is there really no way to run FreeBSD on GitHub Actions? I was pretty happy that we managed to consolidate our CI a while back...

Reply:
Not without major hassle, unfortunately. Like via quemu or something of that sort. But in my experience using Cirrus with other projects it has quite low overhead and is otherwise "set and forget". Examples of some other projects that use both Actions and Cirrus are git itself: https://github.com/git/git/runs/3416539724